test/unit/xml_state_test.py

```diff
@@ -156,8 +163,12 @@ def test_set_repository(self):
         assert self.state.xml_data.get_repository()[0].get_type() == 'type'
         assert self.state.xml_data.get_repository()[0].get_alias() == 'alias'
         assert self.state.xml_data.get_repository()[0].get_priority() == 1
-        assert self.state.xml_data.get_repository()[0].get_imageinclude() is True
-        assert self.state.xml_data.get_repository()[0].get_package_gpgcheck() is False
+        assert self.state.xml_data.get_repository()[
+            0
```

Comment:
[Minor nitpick] I really don't like the formatting here, I think it obscures that the first element of `get_repository()` is taken. (This applies to all the other lines too).

Reply:
:smile: ok, you are right... just a sec

Reply:
how about it now ?

Reply:
Much better, thanks!